src/main/java/uk/gov/ons/fwmt/job_service/rest/impl/JobResourceServiceImpl.java

```diff
@@ -90,4 +104,47 @@ public boolean updateJob(JobDto jobDto) {
     return false;
   }
 
+  @Override
+  public void sendCSV(MultipartFile file) {
+
+    log.info("SendCSV");
+    try {
+
+      File convFile = convertFile(file);
+      Resource fileConvert = new FileSystemResource(convFile);
+
+      MultiValueMap<String,Object> bodyMap = new LinkedMultiValueMap<>();
+      bodyMap.add("file",fileConvert);
+      HttpHeaders headers = new HttpHeaders();
+      headers.setContentType(MediaType.MULTIPART_FORM_DATA);
+
+      final HttpEntity<MultiValueMap<String, Object>> request = new HttpEntity<>(bodyMap, headers);
+      //final HttpEntity<File> request = new HttpEntity<File>(convFile);
+      final ResponseEntity<String> sendCSVResponseEntity = restTemplate.exchange(sendCSVUrl, HttpMethod.POST, request, String.class);
+      sendCSVResponseEntity.getStatusCode().equals(HttpStatus.OK);
+    } catch (HttpClientErrorException httpClientErrorException) {
+      log.error("An error occurred while communicating with the resource service", httpClientErrorException);
+    } catch (FileNotFoundException e) {
+      e.printStackTrace();
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+  }
+
+
+  private File convertFile(MultipartFile file) throws IOException{
+
+    File convFile = new File (file.getOriginalFilename());
+    FileOutputStream fos = new FileOutputStream(convFile);
+    try {
+      convFile.createNewFile();
+      fos.write(file.getBytes());
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
log errors, dont e.printstacktrace().
You caught the exception and and then resent it, not much point.
https://www.amazon.co.uk/Effective-Java-Second-Joshua-Bloch/dp/0321356683
https://www.amazon.co.uk/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882/ref=sr_1_1?s=books&ie=UTF8&qid=1530797215&sr=1-1&keywords=clean+code